config/ols-config/ols-config.yaml

```diff
@@ -1,37 +1,151 @@
+    preferredPrefix: MONDO_HARRISONS
+    title: Mondo Disease Ontology - Harrisons View (Developmental Snapshot)
+    uri: http://purl.obolibrary.org/obo/mondo/mondo-harrisons-view.owl
+    description: >
+      An experimental view of Mondo, showing the ontology with a different upper level organisation.
+      Basically, the top two levels of the ontology are modelled after the Harrisons Textbook,
+      and everything underneath follows the usual Mondo structure. Please do not use - this is only
+      a proof of concept. Contact Nicole Vasilevsky.
+    base_uri:
+      - http://purl.obolibrary.org/obo/MONDO_
+    synonym_property:
+      - http://www.geneontology.org/formats/oboInOwl#hasExactSynonym
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+#    label_property:  http://www.w3.org/2004/02/skos/core#prefLabel
+    reasoner: EL
+    oboSlims: false
+    ontology_purl : file:/mnt/ontologies/mondo-harrisons-view.owl
+  - id: chr
+    preferredPrefix: CHR
+    title: Monochrom Ontology (Development Snapshot)
+    uri: https://raw.githubusercontent.com/monarch-initiative/monochrom/master/chr.owl
+    description: >
+      Inofficial development snapshot of the Human Phenotype Ontologies. https://github.com/monarch-initiative/monochrom/issues
+    base_uri:
+      - http://purl.obolibrary.org/obo/HP_
+    synonym_property:
+      - http://www.geneontology.org/formats/oboInOwl#hasExactSynonym
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+#    label_property:  http://www.w3.org/2004/02/skos/core#prefLabel
+    reasoner: EL
+    oboSlims: false
+    ontology_purl : file:/mnt/ontologies/chr.owl
+  - id: uberon_human_view
+    preferredPrefix: UBERON_HUMAN_VIEW
+    description: >
+      Human View Subset of Uberon
+    title: Uberon (Human View)
+    uri: http://purl.obolibrary.org/obo/uberon/subsets/human-view.owl
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+    reasoner: EL
+    oboSlims: false
+    ontology_purl : file:/mnt/ontologies/uberon-human-view.owl
+  - id: vbo
+    preferredPrefix: VBO
+    description: >
+      Vertebrate Breed Ontology is an ontology created to serve as a single computable resource for vertebrate breed names.
+    title: Vertebrate Breed Ontology
+    uri: https://raw.githubusercontent.com/monarch-initiative/vertebrate-breed-ontology/master/vbo.owl
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+    reasoner: EL
+    oboSlims: false
+    ontology_purl : file:/mnt/ontologies/vbo.owl
+  - id: phenio
+    preferredPrefix: PHENIO
+    description: >
+      An ontology for accessing and comparing knowledge concerning phenotypes across species and genetic backgrounds.
+    title: Phenomics Integrated Ontology
+    uri: https://github.com/monarch-initiative/phenio/releases/latest/download/phenio.owl
+    definition_property:
+      - http://purl.obolibrary.org/obo/IAO_0000115
+    reasoner: EL
```

Comment:
remove this row and see if this sorts out your cycle issue?

Reply:
I attempted to remove lines 147-149 as you mentioned above, but I still got those cycle warnings when importing `phenio`. Were you suggesting just removing those three lines, just the line that says `reasoner: EL`, or were you suggesting removing the entire `phenio` block?

Reply:
Just FYI, I'm currently trying to import phenio after removing all lines in `phenio.owl` that contain `CARO_0001008`, just as a test. It's been running for ~16 hours now, but at least there aren't any cycle warnings in the log. I'm going to let it run for around 24 hours, but do let me know if you have an idea of how long you expect it to take.

Reply:
This is probably not worth it. We need a much faster deployment, so I assume we need a uncycled phenio. You can comment out the hole block and I will go back to the drawing board! Thanks for checking!